django/db/models/sql/subqueries.py

```diff
@@ -90,6 +90,10 @@ def add_update_values(self, values):
                 not (field.auto_created and not field.concrete) or not field.concrete
             )
             model = field.model._meta.concrete_model
+            if field.name == "pk" and model._meta.is_composite_pk:
```

Comment:
I'm open to different ideas here. I looked into:
- `isinstance(field, CompositePrimaryKey)`: introduces circular import
- `hasattr(field, "fields") and len(field.fields) > 1` : the duck typing seemed less self-documenting

Reply:
I like it!